Dockerfile

```
@@ -206,6 +206,22 @@ RUN set -ex; \
         done; \
     fi
 
+ARG PGVECTO_RS
+RUN set -ex; \
+    if [ -n "${PGVECTO_RS}" ]; then \
+        for pg in ${PG_VERSIONS}; do \
+            # Vecto.rs only support PostgreSQL 14 and upwards
+            if [ $pg -gt 13 ]; then \
+                curl --silent \
+                    --location \
+                    --output /tmp/vectors.deb \
+                    "https://github.com/tensorchord/pgvecto.rs/releases/download/v${PGVECTO_RS}/vectors-pg${pg}-v${PGVECTO_RS}-$(uname -m)-unknown-linux-gnu.deb" && \
```

I missed this, they changed the artifact naming:

```suggestion
                    "https://github.com/tensorchord/pgvecto.rs/releases/download/v${PGVECTO_RS}/vectors-pg${pg}_v${PGVECTO_RS}_$(dpkg --print-architecture).deb" && \
```

---

Thanks for the hint, missed this as well.